src/includes/sourcemaps/generate/javascript.mdx

```diff
@@ -1,4 +1,4 @@
-Most modern JavaScript transpilers support source maps. Below are instructions for some common tools.
+Most modern JavaScript transpilers support source maps. Below are instructions for some common tools. Note that the directions below ensure that your source maps can be uploaded to Sentry, but you must generate them.
```

Comment:
Don't we want to say explicitly that Sentry's Webpack plugin doesn't generate the sourcemaps?

Reply:
This works for me! I wasn't sure how to phrase this but didn't want to close this to create an issue if we could just figure it out on the existing PR. @yuval-sentry does this work for you?

Reply:
@yuval-sentry Ah I didn't know you were referring to the webpack plugin. I think we should adjust the webpack section rather than the top of the page. I agree that the wording there is slightly confusing because it implies that the plugin generates source-maps, whereas it's actually the `devtool` option. Let me quickly whip up a commit.

Reply:
Ok, I took the liberty of updating the relevant section. Lmk what you think. Feel also free to make any additional changes!